src/app/mainPage/MainPage.styles.ts

```diff
@@ -0,0 +1,8 @@
+export const styles = {
+  container: {
+    margin: { marginTop: '0' },
+    spacing: 2,
+    columns: { xs: 6, sm: 12, md: 12 },
```

Comment:
Wiesz co, do styli wrzucałbym tylko to, co idzie do propsa `sx`, cała reszta to bardziej konfiguracja propsa, a nie jego stylowanie